apps/console/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # @wso2is/console
 
+## 2.58.4
```

Comment:
Changelog modifications should not be expected I. Normal PRs. It's meant fro changeset bot only.

Reply:
I think this is a PR conflict. Resolved them.